labs/embeddings/base.py

```diff
@@ -23,42 +22,31 @@ def __init__(self, embedder: type["Embedder"], *args, **kwargs):
     def embed(self, prompt, *args, **kwargs) -> Embeddings:
```

Comment:
you should call this file `embedder.py` since that's the only thing that this file has